apps/agent/app/src/App.js

```diff
@@ -1,42 +1,48 @@
-import React from 'react'
-import { Header, Layout, SyncIndicator, useLayout } from '@aragon/ui'
+import React, { useCallback } from 'react'
 import { useAragonApi } from '@aragon/api-react'
-import InstallFrame from './components/InstallFrame'
+import { Header, Main, SyncIndicator, useLayout } from '@aragon/ui'
+import Balances from './components/Balances'
 import { IdentityProvider } from './components/IdentityManager/IdentityManager'
-import ComingSoon from './components/ComingSoon'
+import InstallFrame from './components/InstallFrame'
+import Transactions from './components/Transactions'
 
 function App({ api, appState, isSyncing }) {
```

Comment:
They're being passed as props from the initial `useAragonApi()` call on the `export default () =>` part of the file. :) I'm passing them as props to avoid a duplicated call to the hook. We could, of course, remove the export default and just do everything on the `App` component.

Reply:
Yes that’s what I meant: since we are not really using `api`, `appState` or `isSyncing` in the `export default() =>`, we might as well get them from the hook in `App` :smile: 

IIRC it was originally made this way because `App` was a class.